server/grpc_service.go

```diff
@@ -2478,6 +2478,29 @@ func (s *GrpcServer) ReportMinResolvedTS(ctx context.Context, request *pdpb.Repo
 	}, nil
 }
 
+// GetMinResolvedTS implements gRPC PDServer.
+func (s *GrpcServer) GetMinResolvedTS(ctx context.Context, request *pdpb.GetMinResolvedTSRequest) (*pdpb.GetMinResolvedTSResponse, error) {
+	fn := func(ctx context.Context, client *grpc.ClientConn) (interface{}, error) {
+		return pdpb.NewPDClient(client).GetMinResolvedTS(ctx, request)
+	}
+	if rsp, err := s.unaryMiddleware(ctx, request, fn); err != nil {
+		return nil, err
+	} else if rsp != nil {
+		return rsp.(*pdpb.GetMinResolvedTSResponse), nil
+	}
+
+	minResolvedTS := s.GetClusterMinResolvedTS()
+	var stores []*pdpb.StoreMinResolvedTS
+	if len(request.GetStoresId()) != 0 {
+		minResolvedTS, stores = s.GetMinResolvedTSByStoreIDs(request.GetStoresId())
```

Comment:
minResolvedTS will be covered？

Reply:
it will return min ts among these stores rather than the whole cluster, which is expected?

Reply:
Yes, I think it is better to return scope min ts.

Reply:
We need to add some comments about it, such as kvproto